designs/administrator/4.6/guided-tours/index.md

```diff
@@ -0,0 +1,102 @@
+---
+parent: Administrator
+version: 4.6
+---
+
+# Guided tours
+
+In 4.6, guided tours will be added to the console. This documentation will outline how tours are designed and should behave. The content of the tour will not be detailed here. Content copy can be found [here](https://docs.google.com/document/d/1qEBDNZYaKF4TDogJLYUqX-8xHCmPCKdaHfKLxMlnZQs/edit?usp=sharing), where it will be reviewed and updated as needed.
+
+## Accessing tours
+Tours can be accessed on both the Administrator and Developer perspectives by going to the **Help menu** and selecting **Guided Tours**. Each of the perspectives will also surface links to the tours in other places around the console. For more detail around how these links will be surfaced in the Developer perspective, view the Developer side documentation.
+
+In the Administrator perspective, a badge will appear on the Overview page informing users that guided tours are available. When clicked, the badge will redirect users to the Guided Tours page.
+
+![1](img/admin1.png)
+
+The Guided Tours page will use a catalog-like design with each tour being shown on a tile. Tiles will provide a brief description of the tour, a time estimate for completing the tour, as well as prerequisites for completing the tour, if any.
+
+![2](img/admin2.png)
+
+If a user clicks on a tour tile, the tour side panel will open. The introductory panel will give users more information and allows the user to start the tour using the **Start tour** button.
+
+![3](img/tour1-screen0.png)
```

Comment:
This is mentioned below within steps 2+, but would the overview of all steps here be clickable as well to quickly jump to a specific one?

Reply:
That's a great question. I hadn't considered it but now that you bring it up I think yes? Especially given the use case that a user has started the tour and has come back to finish it. If they click "Restart tour" instead of "Resume tour" I imagine these would been green or blue depending on which steps they've completed (based on whether they used Check your work) and gives them a nice opportunity to jump right back in. What do you think?

Reply:
That behavior makes sense to me! 👍

Reply:
Awesome! Added this at the end of the documentation :)